libraries/lib-strings/Internat.h

```diff
@@ -58,7 +58,7 @@ extern STRINGS_API const wxString& GetCustomSubstitution(const wxString& str1 );
 
    // Eventually pulls in <windows.h> which indirectly defines DebugBreak(). Can't
    // include <windows.h> directly since it then causes "MemoryX.h" to spew errors.
-   #include <wx/app.h>
+   extern "C" __declspec(dllimport) void __stdcall DebugBreak();
```

Comment:
Simple enough, but then, delete the comment that precedes.

Reply:
Hmm, one sec

Reply:
Force pushed without a comment